recipes/wfa2-lib/meta.yaml

```diff
@@ -25,6 +25,8 @@ requirements:
   host:
     - pkg-config
     - rhash  # [osx]
+    - libgomp  # [linux or aarch64]
```

Comment:
IMO this should be just `# [linux]`. It is `true` for both aarch64 and x86_64, so `or aarch64` won't be evaluated anyway.